Source/WebCore/loader/FrameLoader.cpp

```diff
@@ -1561,6 +1561,13 @@ void FrameLoader::loadURL(FrameLoadRequest&& frameLoadRequest, const String& ref
         policyChecker().stopCheck();
         policyChecker().setLoadType(newLoadType);
         RELEASE_ASSERT(!isBackForwardLoadType(newLoadType) || history().provisionalItem());
+
+        RefPtr domWindow = frame->document()->domWindow();
```

Comment:
We may be able to use the protected variant of the domWindow accessor, if there is one.